apps/tailwind-components/pages/Form.story.vue

```diff
@@ -119,10 +125,22 @@ watch(
 watch(
   () => rowIndex.value,
   async () => {
+    const query: { schema: string; table: string; rowIndex?: number } = {
```

Comment:
interestingly, this row select is actually a very good test case for the Ref input. So once we have implemented the 'select' display for the Ref we should replace it.